.circleci/config.yml

```diff
@@ -3,33 +3,33 @@ version: 2
 jobs:
   lint:
     docker:
-      - image: circleci/python:3.7-buster
+      - image: debian:buster
```

Comment:
All these CI config and dependency updates can be pulled out into a separate PR, if you'd prefer—just say the word, @rocodes. Otherwise I'll leave them in as-is.